### PR TITLE
Add authorization

### PR DESCRIPTION
**Pull request by aborov**

### CODE REVIEW

File: app/views/comments/_comment.html.erb [hunk 1]
```diff
@@ -10,13 +10,16 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <% if current_user == comment.author %>
```
Comment (jniewiel) [inline app/views/comments/_comment.html.erb:13]: Great job limiting a comment's edit and delete access exclusively to its author.

File: app/views/photos/_photo.html.erb [hunk 2]
```diff
@@ -7,18 +7,20 @@
     </h2>
 
     <div>
-      <%= link_to edit_photo_path(photo), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <% if current_user == photo.owner %>
```
Comment (jniewiel) [inline app/views/photos/_photo.html.erb:10]: Nice job limiting a photo's edit and delete links only to its owner!

File: config/routes.rb [hunk 3]
```diff
@@ -4,14 +4,14 @@
   devise_for :users
   
   resources :comments
-  resources :follow_requests
-  resources :likes
-  resources :photos
+  resources :follow_requests, except: [:index, :show, :new, :edit]
+  resources :likes, only: [:create, :destroy]
+  resources :photos, except: [:index]
```
Comment (jniewiel) [inline config/routes.rb:9]: Great job with limiting each resource's available routes.

**Review by jniewiel — APPROVED** [review]
Great job, Aleksei!

**Review by aizatibraimova — APPROVED** [review]

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job, Aleksei!

**Pull request merged**